crud-backend/src/index.tsx

```diff
@@ -0,0 +1,60 @@
+import express, { Express, Request, Response, Application } from 'express';``
+// const express = require("express");
```

Comment:
When landing a PR, you want to remove all commented out code. It's unclear to future readers why it's there, and over time the code tends to get outdated.

It's a good practice to proactively review your PR before asking for review to look for any problems like this.